### PR TITLE
Testing/3.2.1.20180327

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,17 +10,20 @@ osx: &osx
    language: generic
 matrix:
    include:
-
-      - <<: *linux
-        env: CONAN_GCC_VERSIONS=4.9 CONAN_DOCKER_IMAGE=lasote/conangcc49
+#      - <<: *linux
+#        env: CONAN_GCC_VERSIONS=4.9 CONAN_DOCKER_IMAGE=lasote/conangcc49
```
Comment (SSE4) [inline .travis.yml]: any reason why 4.9 was commented out?

File: conanfile.py [hunk 2]
```diff
@@ -65,10 +65,10 @@ def configure(self):
         del self.settings.compiler.libcxx
 
     def source(self):
-        download("https://github.com/glfw/glfw/archive/%s.zip" % self.version, "%s.zip" % self.sources_folder)
+        download("https://github.com/glfw/glfw/archive/0a3c4f5d80b041ee1a12c8da3503653d98bd1a15.zip", "%s.zip" % self.sources_folder)
```
Comment (SSE4) [inline conanfile.py]: please avoid duplication of revision, e.g. by adding variable called revision :)

File: test_package/CMakeLists.txt [hunk 3]
```diff
@@ -10,4 +10,3 @@ file(GLOB SOURCE_FILES *.cpp)
 
 add_executable(${PROJECT_NAME} ${SOURCE_FILES})
 target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
-set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
```
Comment (SSE4) [inline test_package/CMakeLists.txt:13]: can you confirm new version doesn't require C++11 any longer?

**Review by SSE4 — CHANGES_REQUESTED** [review]
in general, looks very impressive, just few minor comments

File: .travis.yml (hunk 1 above)
Comment (ghost) [reply .travis.yml]: It has this issue setting up dependencies: https://travis-ci.org/dimi309/conan-glfw/jobs/361244588 I haven't been able to resolve it. This is new. It didn't use to happen beffore.

Comment (SSE4) [reply .travis.yml]: seems like it needs mesa-common-dev, may be add this one into system requirements as well?

File: test_package/CMakeLists.txt (hunk 3 above)
Comment (ghost) [reply test_package/CMakeLists.txt:13]: I wasn't aware that it ever did require it. In any case, I didn't remove this property consciously. I only re-copied the standard bincrafters test CMakeLists over it and I guess that that's how it got removed. And I didn't put it there in the first place, unless I did so uninentionally...

Comment (SSE4) [reply test_package/CMakeLists.txt:13]: okay, thanks for the clarification
as CI builds fine, let's leave this as is for now

File: .travis.yml (hunk 1 above)
Comment (ghost) [reply .travis.yml]: Ok, I'll try that right now

File: conanfile.py (hunk 2 above)
Comment (ghost) [reply conanfile.py]: Sure :) Done!

File: .travis.yml (hunk 1 above)
Comment (ghost) [reply .travis.yml]: It still has the same problem: https://travis-ci.org/dimi309/conan-glfw/jobs/368442657 Maybe mesa-common-dev was being installed anyway, as a dependency of another dependency. Note that this only happens with the gcc 4.9 image.

Comment (SSE4) [reply .travis.yml]: thanks for trying, I will take a look running the same docker image.

Comment (ghost) [reply .travis.yml]: Ok. I think this is somehow related to the fact that we are compiling a 32-bit binary on a 64-bit VM. We have support for cross-compilation but there seems to be a glitch, only on the gcc 4.9  image.

**Comment by SSE4** [comment]
hi again, @dimi309 
somehow your PR got merge conflicts, could you possibly rebase?

**Comment by ghost** [comment]
Fixed!

**Pull request merged**